pkg/kfake/cluster.go

```diff
@@ -353,7 +354,14 @@ func (c *Cluster) KeepControl() {
 	c.keepCurrentControl.Swap(true)
 }
 
-func (c *Cluster) tryControl(kreq kmsg.Request) (kresp kmsg.Response, err error, handled bool) {
+// RequestNodeId is solely valid from within a control function; it returns
+// the broker id that the request was received by.
+func (c *Cluster) RequestNodeId() int32 {
+	return c.currentBroker.Load().node
```

Comment:
Can this return -1 if Load() returns nil? Currently this will panic if called too early